lib/widgets/mission_creation.dart

```diff
@@ -0,0 +1,233 @@
+                          variableName: variableNameController.text,
+                          variableType: variableTypeController.text));
+                  variableNameController.clear();
+                  variableTypeController.clear();
+                },
+                child: const Icon(
+                  Icons.add,
+                  color: eerieBlack,
+                ),
+                backgroundColor: gray,
+              )),
+        )
+      ]),
+    ]);
+  }
+
+  TableRow createTableRow(String name, String type,
+      {Color color = raisingBlack,
+      bool isTop = false,
+      bool isBottom = false,
+      int rowIndice = -1}) {
+    return TableRow(
+        decoration: BoxDecoration(
+          borderRadius: isTop
+              ? const BorderRadius.only(
+                  topLeft: Radius.circular(10), topRight: Radius.circular(10))
+              : isBottom
+                  ? const BorderRadius.only(
+                      bottomLeft: Radius.circular(10),
+                      bottomRight: Radius.circular(10))
+                  : null,
+          color: color,
+        ),
+        children: [
+          GestureDetector(
+            onDoubleTap: () {
+              if (rowIndice != -1) {
+                BlocProvider.of<VariablesBloc>(context)
+                    .add(DeleteVariable(variableIndex: rowIndice));
+              }
+            },
+            child: Padding(
+              padding: const EdgeInsets.fromLTRB(20, 7.5, 12, 7.5),
+              child: Text(
+                name,
+                style: const TextStyle(
+                  fontWeight: FontWeight.normal,
+                  color: white,
+                  fontFamily: 'DMSans',
+                ),
+              ),
+            ),
+          ),
+          GestureDetector(
+            onDoubleTap: () {
+              if (rowIndice != -1) {
+                BlocProvider.of<VariablesBloc>(context)
+                    .add(DeleteVariable(variableIndex: rowIndice));
+              }
+            },
+            child: Center(
+              child: Padding(
+                padding: const EdgeInsets.all(7.5),
+                child: Text(
+                  type,
+                  style: const TextStyle(
+                    fontWeight: FontWeight.w200,
+                    color: white,
+                    fontFamily: 'DMSans',
+                  ),
+                ),
+              ),
+            ),
+          ),
+        ]);
+  }
+
+  BlocBuilder variablesTable() {
+    return BlocBuilder<VariablesBloc, VariablesState>(
+        builder: (context, state) {
+      List list = state.variablesList.getVariablesList();
+      return Column(
+        children: [
+          Table(
+            border: const TableBorder(
+                verticalInside:
+                    BorderSide(width: 1, color: gray, style: BorderStyle.solid),
+                borderRadius: BorderRadius.all(Radius.circular(10))),
```

Comment:
Eu estava tentando rodar aqui e ta dando problema nessa linha em relação ao borderRadius. Dei uma olhada na documentação mas não achei nada falando sobre o borderRadius dentro do TableBorder. 

![Screenshot from 2021-08-23 09-28-25](https://user-images.githubusercontent.com/67326251/130447118-171128b6-1a73-4fa7-86b3-e7ac08eba5c7.png)

Reply:
Boa observação, você poderia dizer qual versão do flutter você está usando? Acho que esse problema é culpa minha, eu esqueci que eu estou na branch **master** e não na **stable** do repositório do flutter. O parâmetro *borderRadius:* foi introduzido recentemente no flutter e acho que nem foi pra branch **stable** ainda. O [PR](https://github.com/flutter/flutter/pull/85946) que adicionaram essa propriedade ainda é muito novo, aparentemente no dia 15/07/2021. O mesmo arquivo da branch **stable** está sem o parâmetro *borderRadius:*, [aqui](https://github.com/flutter/flutter/blob/stable/packages/flutter/lib/src/rendering/table_border.dart).  De qualquer forma, vou voltar para a **stable** para não termos esse problema novamente. Acham que devo alterar o arquivo para ficar compatível ou esperamos que essa alteração chegue na **stable**, já que ela parece ser simples? @jorgemrisco

Reply:
De fato, minha versão do flutter é a Stable. Eu instalei o flutter pelo Snap e pelo jeito ele pega a versão stable por default.
A minha versão do flutter é a seguinte:
>Flutter is already up to date on channel stable
Flutter 2.2.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision f4abaa0735 (8 weeks ago) • 2021-07-01 12:46:11 -0700
Engine • revision 241c87ad80
Tools • Dart 2.13.4

Comentando essa linnha o app continua a funcionar normalmente, conforme mostra a imagem abaixo. Então eu acho que deixar essa linha comentada por enquanto e abrir um issue pra 'descomentar' ela quando essa funcionalidade for inserida na versão stable.
![a](https://user-images.githubusercontent.com/67326251/130489384-9bf02531-7da7-4c62-b84e-6ee240b2c9dc.jpeg)

Reply:
Boa Fran, o *borderRadius* estava redundante, pois as rows já definem um raio também. Acabei retirando o parâmetro da **TableBorder**, ai a gente não precisa se preocupar.